tests/net/net_pkt/prj.conf

```diff
@@ -15,3 +15,7 @@ CONFIG_NET_LOG=y
 CONFIG_ENTROPY_GENERATOR=y
 CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ZTEST=y
+
+# Do not print any errors, they just confuse the sanitychecker
+# even if there is no problem with the test.
+CONFIG_NET_MAX_LOG_LEVEL_OFF=y
```

Comment:
I'm not sure how this is related to mentioned "Fixes #10777".

Reply:
(Or the commit message for that matter either.)

Reply:
The test was printing some errors that were not related to the passing of the test. That is the reasong for the setting.

Reply:
What if later it turns out that errors were useful/interesting, but nobody will have a chance to see them?

P.S. If commit message have a sample of kind of messages which are disabled, there wouldn't be these questions.